stdlib/public/Concurrency/Actor.cpp

```diff
@@ -48,9 +48,12 @@
 #include <sys/syscall.h>
 #endif
 
-#if HAVE_PTHREAD_H
+#if __has_include(<pthread.h>)
 #include <pthread.h>
 #endif
+#if __has_include(<pthread_np.h>)
```

Comment:
I suggest putting this inside the `#if HAVE_PTHREAD_H` block above.

Please add a comment that the `#if __has_include` here should be replaced with `#if HAVE_PTHREAD_NP_H` someday so that people don't proliferate this pattern.

Reply:
Done. But since we are defining `HAVE_PTHREAD_H` from the build system on this platform, we might as well define `HAVE_PTHREAD_NP_H` on the platform too...

Reply:
Unfortunately, `HAVE_PTHREAD_H` is coming from the LLVM configuration which is awkward for us to change.